Sources/BandcampDownloader/Windows/MainWindow.xaml.cs

```diff
@@ -977,46 +913,6 @@ public partial class MainWindow: Window
             Cursor = Cursors.Arrow;
         }
 
-        private void checkBoxResizeCoverArt_CheckedChanged(object sender, RoutedEventArgs e) {
-            if (checkBoxResizeCoverArt == null || textBoxCoverArtMaxSize == null) {
-                return;
-            }
-
-            textBoxCoverArtMaxSize.IsEnabled = checkBoxResizeCoverArt.IsChecked.Value;
-        }
-
-        private void checkBoxRetrieveFilesizes_CheckedChanged(object sender, RoutedEventArgs e) {
-            if (checkBoxRetrieveFilesizes == null) {
-                return;
-            }
-
-            UserSettings.RetrieveFilesizes = checkBoxRetrieveFilesizes.IsChecked.Value;
-        }
-
-        private void checkBoxSaveCoverArt_CheckedChanged(object sender, RoutedEventArgs e) {
```

Comment:
Why is the `CheckedChanged` event removed?

Reply:
See above as well. 😄 
This event isn't used to save the setting to file, but to enable/disable the cover art settings depending on the fact the the cover art is retrieved or not.